politeiad/cmd/politeia/README.md

```diff
@@ -28,6 +29,22 @@ Save to /home/user/.politeia/identity.json or ctrl-c to abort
 Identity saved to: /home/user/.politeia/identity.json
 ```
 
+## Verify a record
+
+Verifies the censorship signature of a record to make sure it was received by
+the server. It receives as input the server's public key, the record token, 
+the record merkle root and the signature.
```

Comment:
Where is the user getting the server public key from?

Reply:
He can get it from the `politeia identity` command. Is that not correct?